### DISCUSSION TITLE
Kaspersky issues warning/notifications when you using internal Chrome

### DISCUSSION THREAD

**Discussion by rafi-d** [FlareSolverr/FlareSolverr#1165] [General]
Hi,
Using the latest Windows 10 here . I am getting constant warnings/notifications on every search with Jackett using this service, issued by Kaspersky - (free 21.9.6.465)  regarding some *.js (uical.min.js) scripts being downloaded (I spotted two such files-blocked till now). Here is an example:

`Event: Download denied
User: NT AUTHORITY\SYSTEM
User type: System user
Application name: chrome.exe
Application path: D:\Program Files\Flaresolverr\chrome
Component: Safe Browsing
Result description: Blocked
Type: Contains adware, auto-dialers, legitimate software that can be used by intruders to damage your computer or personal data
Name: not-a-virus:HEUR:AdWare.Script.Redirect.gen
Precision: Partially
Threat level: Medium
Object type: File
Object name: uical.min.js
Object path: https://d3mr7y154d2qg5.cloudfront.net
MD5 of an object: A58B4439F29A8705B6AE39EC471E2BFB
Reason: Expert analysis
Databases release date: Yesterday, 22/04/2024 4:43:00

`
Any idea how to avoid this?

![image](https://github.com/FlareSolverr/FlareSolverr/assets/2458280/2787fc11-eb14-4c07-9fb2-d75a4ac75f7d)

**Comment by ilike2burnthing** [comment]
What site are you accessing when this happens (restart Jackett and test indexers individually if need be)?

**Comment by rafi-d** [comment]
https://extratorrent.st/

![image](https://github.com/FlareSolverr/FlareSolverr/assets/2458280/a99b6d76-3f29-497f-9ff2-ec7835ee065e)

![image](https://github.com/FlareSolverr/FlareSolverr/assets/2458280/1c0e1a81-c6f9-473e-aef5-e2c72e4b6b7c)

**Reply by ilike2burnthing** [reply]
If you access that site in Chrome without ad blockers or content filters, do you get the same alert from Kaspersky?

**Comment by rafi-d** [comment]
Just tried, and after disabling ADP and uBlock origin - yes , similar alerts . When I re-enable those addons - the warnings stop. 
I guess this means that there is not much we can do about it?

**Reply by ilike2burnthing** [reply]
Yea, it just some dodgy ad on the site, and Kaspersky is blocking it, you're fine.

As a side note, just stick with uBlock Origin. Using additional blockers just increases your chances of being detected as using ad blockers, will use additional resources, and won't block anything additional.
